.circleci/config.yml

```diff
@@ -20,6 +20,22 @@ jobs:
           covermode: atomic
           failfast: true
           race: true
+  golang-lint-test-validation:
+    working_directory: ~/superchain-registry/validation
+    executor:
+      name: go/default  # is based on cimg/go
+      tag: '1.20'
+    steps:
+      - checkout:
+          path: ~/superchain-registry
+      - go/load-cache
+      - go/mod-download
+      - go/save-cache
+      - run: golangci-lint run -c=.golangci.yml
```

Comment:
I think we should extract the linting into its own job that lints every project at once. Makes it easier to identify failures since lints won't block the tests from running.

Reply:
[4a43a41](https://github.com/ethereum-optimism/superchain-registry/pull/62/commits/4a43a41ef4b1234f4d896eebd6bd63eb96dbab8d)

Reply:
I also tidied up the testing part of the CI pipeline [eb63f1a](https://github.com/ethereum-optimism/superchain-registry/pull/62/commits/eb63f1a565ef87d00317052c72398cb238f7398b). This will mean the two module's tests won't run concurrently, but IMO this is acceptable.